### DISCUSSION TITLE
Support Range Bar Chart

### DISCUSSION THREAD

**Discussion by buiduclam09** [patrykandpatrick/vico#796] [Ideas]
I can't find it here.

**Comment by Gowsky** [comment]
Hi @buiduclam09, I'm closing the discussion as a duplicate of #430. You can already implement them using candlestick charts, like described [here](https://github.com/patrykandpatrick/vico/discussions/430#discussioncomment-9619819).

**Reply by buiduclam09** [reply]
oh, thank you so much.
